Gopkg.lock

```diff
@@ -136,12 +136,12 @@
   branch = "master"
   name = "github.com/giantswarm/k8scloudconfig"
   packages = [
-    "v_2_0_0",
```

Comment:
I'm curious, where does this change come from?

Reply:
Yesterday I changed legacy `awsconfig/v1` from `v_2_0_0` to `v_2_0_2` for the kube-proxy fix. Not sure why this didn't change then but I think this is OK.

See https://github.com/giantswarm/aws-operator/pull/766/files